node/src/chain_spec/koi.rs

```diff
@@ -132,7 +132,7 @@ pub fn local_config() -> ChainSpec {
 		None,
 		Some(properties()),
 		Extensions {
-			relay_chain: "paseo-local".into(), // You MUST set this to the correct network!
+			relay_chain: "rococo-local".into(), // You MUST set this to the correct network!
```

Comment:
Please confirm this change again. Why not connecting to the latest relay chain?

Reply:
Local tests are always rococo.